Description: Use relative paths in the generated jpms.args files to make the builds reproducible
Author: Emmanuel Bourg <ebourg@apache.org>
Bug: https://issues.apache.org/jira/browse/MCOMPILER-397
--- a/src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java
+++ b/src/main/java/org/apache/maven/plugin/compiler/AbstractCompilerMojo.java
@@ -1078,7 +1078,7 @@
                 if ( value == null )
                 {
                     entry = entryIter.next();
-                    value = entry.getKey();
+                    value = entry.getKey().replaceAll( new File("").getAbsolutePath(), "." );
                 }
                 jpmsLines.add( value );
             }
